### DISCUSSION TITLE
Battery indicator for RG280v

### DISCUSSION THREAD

**Discussion by Mk6rabbit** [eduardofilo/RG350_adam_image#154] [Q&A] [answered]
I was guided to this page to bring up a hopefully minor issue. The battery indicator does not show battery levels correctly, it will show charging while not charging, or low battery life after having been charged. These are just examples. Obviously it may be a hardware issue with my battery but after looking into it, it seems I’m not the only one experiencing this issue

**Comment by eduardofilo** [comment, marked as answer]
Created FAQ number 11 to answer this question.

https://github.com/eduardofilo/RG350_adam_image/wiki/En:-7.-FAQ#q11-why-does-not-show-battery-levels-correctly-the-battery-indicator
